src/encoded/commands/deploy.py

```diff
@@ -696,10 +696,13 @@ def main():
         print(f'\nDry Run')
         print(f'run_args dict keys: {run_args.keys()}')
         print(f'\nRun Variables.  In /etc/environment on instance')
+        run_vars = {}
         for line in run_args['user_data'].split('\n'):
             line = line.strip()
             if line[:5] == 'ENCD_':
-                print(line)
+                key_val = line.split('=')
+                run_vars[key_val[0]] = key_val[1]
+                print(f"{key_val[0]}={run_vars[key_val[0]]}")
```

Comment:
This would be more legible with different unpacking:
```suggestion
                key, value, *_ = line.split('=')
                run_vars[key] = value
                print(f"{key}={value}")
```

Reply:
That's neat unpacking.  If this ends up in final refactor I'll use it.